plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/editors/featuremodel/actions/CreateFeatureAboveAction.java

```diff
@@ -98,12 +98,12 @@ protected boolean isValidSelection(IStructuredSelection selection) {
 			}
 		}
 
-		if ((this instanceof ActionAllowedInExternalSubmodel) || !hasExternalFeature(selection)) {
+		if ((this instanceof ActionAllowedInExternalSubmodel) || isExternalRootFeature(selection)) {
```

Comment:
The name "isExternalRootFeature" is misleading here as the method returns true if the feature is not external.